#Save configuration types
# save_configuration_t - enumeration
uint8 value

#Pre-Defined ENUM Constants
	uint8 configuration_Not_Save     = 0
	uint8 configuration_Save         = 1
	uint8 configuration_Bad_Checksum = 4
